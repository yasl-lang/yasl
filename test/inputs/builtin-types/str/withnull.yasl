const x = 'abc\0def'

echo len x

echo x

echo len x

echo x[4:]

echo len x
