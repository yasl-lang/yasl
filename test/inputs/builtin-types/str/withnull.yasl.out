7
abcdef
7
def
7
